text/0004-recoverable-transaction-history.md

```diff
@@ -148,13 +150,14 @@ Three new memo types are specified:
 
 ## 0x0100 Authenticated Sender Memo
 
-The 44-byte memo data is laid out as follows:
+The 64-byte memo data is laid out as follows:
```

Comment:
It seems odd to adjust the older MCIP to the newest values. That makes it necessary to look at git history to grok the protocol evolution. I believe the link at the top is sufficient, maybe with a similar note here, but not changing this table -- instead duplicate it in the new MCIP.

Reply:
@jcape Could you weigh in on this also? Do you have a preference?

Reply:
The standard in rust appears to be to add a link to the update but not actually change the older file except to note that it has been superceded: https://github.com/rust-lang/rfcs/pull/2050 . I think remoun's suggestion is more consistent with this.